compute_tools/src/pg_helpers.rs

```diff
@@ -196,28 +196,42 @@ impl Escaping for PgIdent {
     }
 }
 
+impl Escaping for &'static str {
+    fn pg_quote(&self) -> String {
+        let result = format!("\"{}\"", self.replace('"', "\"\""));
+        result
+    }
+}
+
```

Comment:
Could we use PgIdent instead of defining this for any string?

I'm worried that this could cause confusion on quoting string literals (i.e. `'` -> `''`) and SQL identifiers (`"` -> `""`)